lib/commonBindings.js

```diff
@@ -66,7 +66,7 @@ function sendConfigurationToDevice(device, apiKey, deviceId, results, callback)
     transportSelector.applyFunctionFromBinding(
```

Comment:
lib/commandHandler.js doesn't needs modification in the IOTA-UL case? (in https://github.com/telefonicaid/iotagent-json/pull/675/files it appears)

Reply:
That file does not exists in iotagent-ul

Reply:
NTC